cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,170 @@
 /// <reference types='cypress' />
 
+import WebTables from '../support/pageObjects/pages/webTables';
+import RegistrationForm from
+  '../support/pageObjects/components/registrationForm';
+import Pagination from '../support/pageObjects/components/pagination';
+import generateWorker from '../support/generateData';
+import workersData from '../support/fixtures/workersData.json';
+import titles from '../support/fixtures/titles.json';
+
+const webTables = new WebTables();
+const registrationForm = new RegistrationForm();
+const pagination = new Pagination();
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.wrap(generateWorker()).as('workerData');
```

Comment:
1. move all aliases to constants 
2. not sure if we need this wrap, we can reduce amount of code by not using wrap, but directly this function. For this, we need just to make sure that function returns something